### PR TITLE
run rustfmt on compiletest folder in src/tools/ 

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/tools/compiletest/src/common.rs [hunk 1]
```diff
@@ -35,43 +35,44 @@ impl FromStr for Mode {
...
-            DebugInfoGdb => "debuginfo-gdb",
-            DebugInfoLldb => "debuginfo-lldb",
-            Codegen => "codegen",
-            Rustdoc => "rustdoc",
-            CodegenUnits => "codegen-units",
-            Incremental => "incremental",
-            RunMake => "run-make",
-            Ui => "ui",
-        }, f)
+                              CompileFail => "compile-fail",
```
Comment (GuillaumeGomez) [inline src/tools/compiletest/src/common.rs]: This change is completely wrong.

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to reopen with comments addressed!

**Pull request closed without merging**